.github/workflows/generate_catalog_templates.yml

```
@@ -0,0 +1,19 @@
+name: "Generate catalog templates (POT files for Transifex)"
```

Could we get smaller names as this is being used for CI and github status :p
```suggestion
name: L10n
```